src/ASM/LR/ASMLRSpline.C

```diff
@@ -307,6 +307,26 @@ IntVec ASMunstruct::getFunctionsForElements (const IntVec& elements)
   return result;
 }
 
+IntVec ASMunstruct::getBoundaryNodesCovered (const IntVec& nodes) const
+{
+  geo->generateIDs();
+  IntVec oneBoundary, result;
+  int numbEdges = (getNoParamDim() == 2) ? 4 : 6;
+  for(int edge=1; edge<=numbEdges;  edge++)
```

Comment:
space between for and (

Reply:
Du har ikke korrigert dette (samt for alle ditto)?? Dessuten dobbel space før `edge++`